### PR TITLE
file should stay near dir

### PR DESCRIPTION
**Pull request by bluelovers**

### CODE REVIEW

**Comment by bluelovers** [comment]
hmm i don't know how to update snapshot

**Comment by fisker** [comment]
@bluelovers `npm run test -u`

File: package.json [hunk 1]
```diff
@@ -17,20 +17,21 @@
...
-  "types": "index.d.ts",
-  "bin": "cli.js",
   "scripts": {
     "lint": "eslint .",
     "semantic-release": "semantic-release",
     "sort-package-json": "node cli.js package.json --check",
     "test": "ava",
-    "test-coverage": "nyc ava"
+    "test-coverage": "nyc ava",
+    "update-snapshot": "ava -u && node cli.js package.json"
```
Comment (fisker) [inline package.json]: I dont think we need this, its on `ava` docs

Comment (bluelovers) [reply package.json]: but this also help when update sort order, can more easy update package.json and snapshot at same time

Comment (keithamus) [reply package.json]: I like this. I’d probably forget `-u` and this is a nice reminder.

**Pull request merged**